src/main/scala/by/itechart/tutorial/dao/Repositories.scala

```diff
@@ -39,12 +34,14 @@ class UserToGroupRepository @Inject()(@Named("db") db: Database) extends Postgre
   def pathAction(userId: Long, groupId: Long): FixedSqlAction[Int, NoStream, Effect.Write] = {
     table insertOrUpdate (UserToGroupEntity(Option.empty, userId, groupId))
   }
-
 }
 
-class UserToUserRepository @Inject()(@Named("db") db: Database) extends PostgresRepository[UserToUserTable, Long](db) {
+class UserToUserRepository @Inject()(
+                                      @Named("db") val db: Database,
+                                      @Named("profile") val profile: JdbcProfile
```

Comment:
Another alternative could be using something like https://docs.scala-lang.org/tour/self-types.html
But not sure which one is better